package.json

```
@@ -23,14 +23,16 @@
     "typescript": "^3.3.1"
   },
   "scripts": {
-    "postinstall": "lerna bootstrap && npm run -s update-greenkeeper-json",
+    "postinstall": "npm run -s update-package-lock && npm run -s update-greenkeeper-json",
+    "ci": "npm ci --ignore-scripts && npm run -s postci",
+    "postci": "lerna bootstrap --ci",
     "prerelease": "npm run build:full && npm run mocha && npm run lint",
     "release": "lerna version && lerna publish from-git --yes",
-    "update-package-lock-json": "lerna exec \"npm install -s --ignore-scripts --package-lock-only --no-audit\" && git add \"**/package-lock.json\"",
+    "update-package-lock": "lerna clean && lerna bootstrap --no-ci",
```

Since this command is updating many package locks, I am proposing to rename it accordingly.

```suggestion
    "update-package-locks": "lerna clean && lerna bootstrap --no-ci",
```